bat/tests/01-create-simple-mix/Makefile

```diff
@@ -3,7 +3,7 @@
 check:
 	bats ./run.bats
 
-CLEANDIRS = ./update ./test-chroot ./logs ./.repos ./bundles ./update ./mix-bundles ./clr-bundles ./local ./results ./repodata ./rpms ./upstream-bundles ./local-bundles
+CLEANDIRS = ./update ./test-chroot ./logs ./.repos ./bundles ./update ./mix-bundles ./clr-bundles ./local-yum ./results ./repodata ./local-rpms ./upstream-bundles ./local-bundles
```

Comment:
Long lines can be split in a couple of different ways, depending on if you want to override them from the command line when invoking `make`. Either you can introduce backslash newline (which gets replaced with a space by make, or you can say
```
CLEANDIRS = ./update ./test-chroot
CLEANDIRS += ./logs
```
The latter although popular, means that invoking `make CLEANDIRS='fred'` will still append ./logs onto it, so care should be taken to decided is CLEANDIRS is part of the api this makefile is presenting to the world.